packages/path_provider/path_provider/README.md

```diff
@@ -19,3 +20,18 @@ String appDocPath = appDocDir.path;
 ```
 
 Please see the example app of this plugin for a full example.
+
+### Usage in tests
+
+Recently `path_provider` was updated to be a federated plugin. 
+
+With that change, tests should be updated to mock `PathProviderPlatform` rather than `PlatformChannel`. 
+
+See this `path_provider` [test](https://github.com/flutter/plugins/blob/master/packages/path_provider/path_provider/test/path_provider_test.dart) for an example.
+
+You will also have to temporarily add the following line to the setup of your test.
+```dart
+disablePathProviderPlatformOverride = true;
+```
+
+See this [issue](https://github.com/flutter/flutter/issues/52267), and these comments [1](https://github.com/flutter/plugins/pull/2789#issuecomment-632354400), [2](https://github.com/flutter/plugins/pull/2789#discussion_r430634873) for details on why this is needed.
```

Comment:
The PR discussions have a lot going on in them, and I also don't really want to encourage people to comment in those PRs forever. I've added a bit more summary to the issue; let's just link to that.